proto-public/pbmulticluster/v2beta1/computed_exported_services.proto

```diff
@@ -11,17 +11,17 @@ import "pbresource/resource.proto";
 message ComputedExportedServices {
   option (hashicorp.consul.resource.spec) = {scope: SCOPE_PARTITION};
 
-  repeated ComputedExportedService consumers = 1;
+  repeated ComputedExportedService services = 1;
 }
 
 message ComputedExportedService {
   hashicorp.consul.resource.Reference target_ref = 1;
-  repeated ComputedExportedServicesConsumer consumers = 2;
+  repeated ComputedExportedServiceConsumer consumers = 2;
 }
 
-message ComputedExportedServicesConsumer {
+message ComputedExportedServiceConsumer {
   // no sameness group
-  oneof consumer_tenancy {
+  oneof tenancy {
```

Comment:
The type being prefixed onto the field name here causes stutter when reading through code since you wind up seeing a lot of `consumer.ConsumerTenancy`.